### ISSUE TITLE
Unable to set theme due to python-ruamel-yaml upgrade

### ISSUE THREAD

**Issue by shootheart** [JezerM/web-greeter#74]
## Environment
- OS: Arch Linux
- web-greeter version: 3.5.2

## Bug description
There was an error when config theme after python-ruamel-yaml upgrade to 0.18.5.

Log as follow:
```
2024-03-16 11:29:18 [ ERROR ] config - config.py:209 : load_config | Config was not loaded:
	
"safe_load()" has been removed, use

  yaml = YAML(typ='safe', pure=True)
  yaml.load(...)

instead of file "/usr/lib/web-greeter/config.py", line 207

            web_greeter_config["config"] = yaml.safe_load(file)


acpi_listen: can't open socket /var/run/acpid.socket: No such file or directory
2024-03-16 11:29:18 [ DEBUG ] Greeter - Greeter.py:116 : __init__ | LightDM API connected
2024-03-16 11:29:18 [ WARNING ] config - config.py:180 : load_theme_config | Theme config was not loaded:
	
"safe_load()" has been removed, use

  yaml = YAML(typ='safe', pure=True)
  yaml.load(...)

instead of file "/usr/lib/web-greeter/config.py", line 176

            theme_config = yaml.safe_load(file)


2024-03-16 11:29:18 [ DEBUG ] config - config.py:181 : load_theme_config | Using default theme config
2024-03-16 11:29:18 [ DEBUG ] browser - browser.py:185 : create_windows | Browser Window created
2024-03-16 11:29:18 [ DEBUG ] screensaver - screensaver.py:57 : set_screensaver | Screensaver timeout set
2024-03-16 11:29:18 [ DEBUG ] browser - browser.py:248 : init | Initializing Browser Window
2024-03-16 11:29:18 [ DEBUG ] browser - browser.py:268 : load_theme | Theme loaded
2024-03-16 11:29:19 [ DEBUG ] browser - browser.py:230 : show | Web Greeter started win: 8601
2024-03-16 11:29:19 [ DEBUG ] browser - browser.py:234 : run | Web Greeter started
```
## Steps to reproduce
Steps to reproduce the behavior.

**Comment by JezerM** [comment]
This has been already fixed in the master branch.

**Comment by rulatir** [comment]
There hasn't been a release in months. Will this fix be unavailable to non-hackers indefinitely?

**Comment by tuurep** [comment]
Jezer if you need a hand for example in co-maintenance of the AUR package, let me know :)

**Comment by JezerM** [comment]
Hi, sorry for the delay. I just published the 3.5.3 version in releases with this fix, and also updated the AUR package.
